packages/vscode-extension/WHATISNEW.md

```diff
@@ -6,6 +7,20 @@
 - [2.8.0 - Oct 18 2021](#280---oct-18-2021)
 - [2.7.0 - Sep 17 2021](#270---sep-17-2021)
 
+## 3.1.2 - Jan 10, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+New Features:
+- Use Service Principle to login Azure account in CICD template.
+- Support building React Tab app by different environment variables for multiple environments.
+
+Enhancement:
+- Improve the debugging experience on WSL.
+- Support .NET SDK 6.0.
+- Speed up NPM installation.
+- Improve the experience to preview manifest file and update manifest file to Developer Portal.
```

Comment:
CLI commands update should be the enhancement or a feature? @MuyangAmigo 
teamsfx manifest validate
teamsfx manifest update